app/models/ability.rb

```diff
@@ -0,0 +1,19 @@
+class Ability
+  include CanCan::Ability
+
+  def initialize(user)
+    user ||= User.new # guest user (not logged in)
+
+    if user.admin?
+      can :manage, :all # admin can do anything
+    else
+      can :read, :all # guest user can read everything
+      can :create, Post
+      can :update, Post, user_id: user.id
+      can :destroy, Post, user_id: user.id
+      can :create, Comment
+      can :update, Comment, user_id: user.id
+      can :destroy, Comment, user_id: user.id
```

Comment:
- **[OPTIONAL]** We can use `:manage` to represent any action. If you had used `:manage` you wouldn't need to define this separate action since `:manage` is a catch-all. You can check more details about this topic [here](https://www.educative.io/answers/how-to-define-abilities-in-cancan)

Reply:
Hi @franclobo ,
Thank you for the suggestion, it's easier and more readable with :manage